api-service/src/configs/DatasetConfigDefault.ts

```diff
@@ -32,12 +32,11 @@ export const defaultMasterConfig = {
     "dataset_config": {
         "data_key": "",
         "timestamp_key": "",
-        "exclude_fields": [],
```

Comment:
Check  with manju

Reply:
Confirmed with manju, exclude fields parameter not required to default